### ISSUE TITLE
Breaking changes to the GraphQL API

### ISSUE THREAD

**Issue by jeffsee55** [tinacms/tinacms#2846]
Hey everyone 👋 

We'll soon be shipping a breaking change to the shape the GraphQL API. One important thing to note: this will _NOT_ be marked as a "major" release. We're staying under `1.0.0` until Tina is out of beta, so please keep in mind that breaking changes like this can happen, though we'll always do our best to communicate them clearly. This will be released in the next day or 2 so please stay tuned!

You can test out the tagged release now by updating `tinacms` and `@tinacms/cli` to `0.0.0-2022321235846` (note that it will only work locally until it's released).

TLDR -

1. The `schema` from `.tina/schema` _must_ be provided to `<TinaCMS>` as seen here https://github.com/tinacms/tina-cloud-starter/blob/simpler-gql-names/.tina/schema.ts#L357-L359
2. The GraphQL API will be much less verbose (notice the query field name and no "data" property):
```graphql
# old
{
  getPostDocument(relativePath: $relativePath) {
    data {
      title
    }
  }
}

# new
{
  post(relativePath: $relativePath) {
    title
  }
}
```
Please have a read through the changes here
https://github.com/tinacms/tinacms/pull/2823

**Comment by jeffsee55** [comment]
This has been released: `"@tinacms/cli": "^0.60.14"` and `"tinacms": "^0.68.0"`
